lib/screens/catalog.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:hometask_v_3/models/cart_methods.dart';
+import 'package:hometask_v_3/models/catalog_list.dart';
+
+class MyCatalog extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: CustomScrollView(
+        slivers: [
+          _MyAppBar(),
+          const SliverToBoxAdapter(child: SizedBox(height: 12)),
+          SliverList(
+            delegate: SliverChildBuilderDelegate(
+                (context, index) => _MyListItem(index)),
```

Comment:
Maybe consider renaming **_MyListItem** to something more self-explanatory.

Reply:
Thanks

Reply:
Try to name variables of the small size for readability